components/pedigree/resources/src/main/resources/pedigree/view/personVisuals.js

```diff
@@ -1100,7 +1102,7 @@ define([
          */
         getAllGraphics: function($super) {
             //console.log("Node " + this.getNode().getID() + " getAllGraphics");
-            return $super().push(this.getHoverBox().getBackElements(), this.getLabels(), this._linkArea, this.getCarrierGraphics(), this.getEvaluationGraphics(), this.getHoverBox().getFrontElements());
+            return $super().push(this.getHoverBox().getBackElements(), this.getLabels(), this._awLabel, this._linkArea, this.getCarrierGraphics(), this.getEvaluationGraphics(), this.getHoverBox().getFrontElements());
```

Comment:
This works here but is not right, as may cause other bugs. Instead, `this.getLabels()` should include `this._awLabel`. Other code may also expects all labels to be set in `getLabels()` (e.g. code to move labels under certain conditions)